packages/expo-device/src/Device.ts

```diff
@@ -0,0 +1,130 @@
+import { Platform, UnavailabilityError } from '@unimodules/core';
+import ExpoDevice from './ExpoDevice';
+
+import { deviceNamesByCode } from './DeviceNameByCode';
+import { DeviceType as _DeviceType } from './Device.types';
```

Comment:
Because in typescript exporting an imported enum types would have different syntax, so if exporting them with the same name, it wouldn't work. Referenced from https://stackoverflow.com/questions/47701270/re-export-typescript-enum-from-namespace.